### ISSUE TITLE
Contributing doc has broken link

### ISSUE THREAD

**Issue by dthaler** [openenclave/openenclave#2727]
https://github.com/openenclave/openenclave/blob/master/docs/Contributing.md says:

> Also see our [Bors dashboard](https://oe-bors.westus2.cloudapp.azure.com/).

However the link to https://oe-bors.westus2.cloudapp.azure.com/ fails.  Chrome reports 
"This site can’t be reached
oe-bors.westus2.cloudapp.azure.com’s server IP address could not be found."

**Comment by jhand2** [comment]
@BRMcLaren is there a new bors dashboard link?

Also, is there a public place where people can see the bors queue (i.e. bors hasn't picked up my bors try yet, can I see the queue of other running bors tries to see when mine will come up?)

**Comment by radhikaj** [comment]
Please investigate if bors try can run in parallel if there are sufficient resources available.
@BRMcLaren Please change the link that @dthaler called out. There should be a way for developers to figure out what is in the queue and it should be documented in the contributing docs.

**Comment by CodeMonkeyLeet** [comment]
Updated link should be https://app.bors.tech/repositories/21855

**Comment by BRMcLaren** [comment]
@jhand2 

Yes, there is a new dashboard link.

Yes, you can see a queue here: https://oe-jenkins-dev.westeurope.cloudapp.azure.com. If you submit multiple PR's, one can see them being queued up under "Build Queue". A frustration I see is not being able to go CI/CD - > PR to understand which is which, but we are actually moving away from this model

@radhikaj 

We are looking at moving towards a model of abstracting out Bors to only manage merge requests and allowing the testing logic to be split up between platforms such as https://github.com/openenclave/openenclave/pull/2703. This would be managed by Github status checks and should solve any intermittent "Bors dying" issues.

A proposal is coming soon for the community to take a look at, I would like to break this off to @dthaler's original issue of the broken link and look into parallelizing Bors if the community decides this is the right route at a later time.

Is that acceptable?

**Comment by dthaler** [comment]
Thanks Brett!  
I agree parallelizing Bors is a separate issue.  This issue is purely about documentation.
